components/Tabs/Tabs.js

```diff
@@ -2,50 +2,62 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = document.querySelector(`.tabs-link[data-tab='${element.dataset.tab}']`);
+    // console.log(this.data);
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+    this.itemElement = document.querySelector(`.tabs-item[data-tab='${element.dataset.tab}'`)
```

Comment:
you're actually doing double work here. this reference being stored in itemElement is correct, but your declaration of `this.data` just wanted the data attribute itself, not going back to the DOM and pulling the element that had the matching dataset. so like this: `this.data = this.element.dataset.tab` because then on this line here, you can use `${this.data}` instead of pulling it off the element again